plugins/org.yakindu.sct.generator.java/src/org/yakindu/sct/generator/java/JavaExpressionsGenerator.xtend

```diff
@@ -149,6 +156,16 @@ class JavaExpressionsGenerator extends ExpressionsGenerator {
 			Declaration: return codeDeclaration(ref, it).toString
 		}
 	}
+		
+	def dispatch String code(MetaCall it) {
+		feature.metaCode(owner)
+	}
+	
+	def dispatch String metaCode(EObject it, Expression exp)
```

Comment:
Why not structure metaCode like in C code generator? Different generator would be more aligned with the option of common core implementations

Reply:
Let's consolidate and harmonize the expression generators in a separate PR.